internal/scorecard/alpha/tests/olm.go

```diff
@@ -132,6 +132,29 @@ func StatusDescriptorsTest(bundle registry.Bundle) scapiv1alpha2.ScorecardTestRe
 	r.State = scapiv1alpha2.PassState
 	r.Errors = make([]string, 0)
 	r.Suggestions = make([]string, 0)
+	csv, err := bundle.ClusterServiceVersion()
```

Comment:
Note that it is not passing in the CI: https://travis-ci.org/github/operator-framework/operator-sdk/jobs/682067742#L923

The CI is complaining about duplicate code which usually indicates that the code is missing encapsulation.